### PR TITLE
Ruler: Add status tab to UI

### PR DESCRIPTION
**Pull request by mjd95**
## Changes

Added status tab to Ruler UI as per https://github.com/thanos-io/thanos/issues/1282.  Just copy-pasted the same for the Query UI.

## Verification
Screen Shot 2019-08-16 at 11 20 04

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Accepted into CNCF:
 - [#1297](https://github.com/thanos-io/thanos/pull/1297) Thanos Compact added `/-/ready` and `/-/healthy` endpoints to Thanos compact.
 - [#1431](https://github.com/thanos-io/thanos/pull/1431) Thanos Query added hidden flag to allow the use of downsampled resolution data for instant queries.
 - [#1408](https://github.com/thanos-io/thanos/pull/1408) Thanos Store Gateway can now allow the specifying of supported time ranges it will serve (time sharding). Flags: `min-time` & `max-time`
+- [#1427](https://github.com/thanos-io/thanos/pull/1427) Added "Status" tab to Ruler UI
```
Comment (bwplotka) [inline CHANGELOG.md]: It has to be part of `Unreleased` now ):

Comment (mjd95) [reply CHANGELOG.md]: 👍 fixed :)

**Comment by daixiang0** [comment]
@mjd95 do you have time to rebase?

**Comment by mjd95** [comment]
Sorry, forgot about this.  @daixiang0 @bwplotka have rebased again.

**Comment by daixiang0** [comment]
@mjd95 could you rebase again :(

**Pull request closed without merging**